google-cloud-core/src/test/java/com/google/cloud/PolicyV3Test.java

```diff
@@ -46,24 +46,22 @@
       ImmutableList.of(USER, SERVICE_ACCOUNT, ALL_USERS);
   private static final List<String> MEMBERS_LIST_2 =
       ImmutableList.of(ALL_AUTH_USERS, GROUP, DOMAIN);
+
+  private static final Binding VIEWER_BINDING =
+      Binding.newBuilder().setRole(VIEWER).setMembers(MEMBERS_LIST_1).build();
+  private static final Binding EDITOR_BINDING =
+      Binding.newBuilder().setRole(EDITOR).setMembers(MEMBERS_LIST_2).build();
   private static final List<Binding> BINDINGS_NO_CONDITIONS =
```

Comment:
```suggestion
  private static final ImmutableList<Binding> BINDINGS_NO_CONDITIONS =
```

`ImmutableList` and friends are especially useful as parameter, method, field, and local variable types, so that users know they are immutable. (For example, the mutation methods are all deprecated, so IDEs will warn about their use.) Please do this throughout the file.

Reply:
done